Composer/packages/extensions/adaptive-form/src/components/fields/NumberField.tsx

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import React from 'react';
-import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
 import { FieldProps } from '@bfc/extension';
 import formatMessage from 'format-message';
-
+import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
+import React from 'react';
 import { FieldLabel } from '../FieldLabel';
 
+const floatNumberOfDecimals = 1;
```

Comment:
Is there a reason we're fixing this at 1? I feel like users might have different use cases where they might need other numbers of decimals (e.g. if they're implementing a "currency" field and need 2, or if they need even more precision).

Reply:
Agreed. It was set up to a precision of 1 so I didn't want to break anything since I am still new to the codebase. How do we make that decision? If we don't have clear guidance then I agree 2 is a better default.